example/provider-local/garden/base/patch-controller-registrations.yaml

```diff
@@ -13,3 +13,11 @@ metadata:
 spec:
   deployment:
     policy: Always
+---
+apiVersion: core.gardener.cloud/v1beta1
+kind: ControllerRegistration
+metadata:
+  name: networking-cilium
+spec:
+  deployment:
+    policy: Always
```

Comment:
I'm not sure why we have introduced this, but I don't think we need it (only for `provider-local` since images are built on the fly). Drop it?

Reply:
Addressed with https://github.com/gardener/gardener/pull/7664/commits/57da282be02f4408a9c52b0680cb95ed3913048d.